scripts/environment/Dockerfile

```
@@ -1,11 +1,16 @@
 FROM docker.io/ubuntu:20.04 AS ENVIRONMENT
+
+# implicitly set by Docker BuildKit
+ARG TARGET_ARCH
+
 ENV DEBIAN_FRONTEND=noninteractive \
     TZ='America/New York' \
     PATH=/root/.cargo/bin:/root/.local/bin/:$PATH \
     LANG=en_US.UTF-8 \
     LANGUAGE=en_US.UTF-8 \
     LC_ALL=en_US.UTF-8 \
-    CROSS_DOCKER_IN_DOCKER=true
+    CROSS_DOCKER_IN_DOCKER=true \
+    TARGET_ARCH=${TARGET_ARCH}}
```

```suggestion
    TARGET_ARCH=${TARGET_ARCH}
```

However I don't think that's sufficient, it looks like Looks like `TARGET_ARCH` is null in `scripts/environment/bootstrap-ubuntu-20.04.sh` based on Jesse's error message